packages/ui/src/typography/TypographyOverrides.ts

```diff
@@ -122,4 +123,15 @@ export function createTypographyOptions(
 
 export function overrideTypography(theme: SuperDispatchTheme): void {
   theme.props.MuiTypography = { variant: 'body2' };
+
+  theme.overrides.MuiTypography = {
+    root: {
```

Comment:
Can we override colorError field instead of css?

Reply:
@doniyor2109 Didn't know about this! Updated